app/controllers/bookingsync_portal/admin/rentals_controller.rb

```diff
@@ -9,7 +9,7 @@ def index
         @visible_rentals = current_account.rentals.visible
         @remote_accounts = current_account.remote_accounts
         @remote_rentals_by_account = current_account.remote_rentals.ordered
-          .includes(:remote_account).group_by(&:remote_account)
```

Comment:
@Mandaryn you cant just override a single method in a controller
You either take from portal or rewrite all.

We handled it the same way for rentals with `visible` scope just above